acceptance/tests/resource/service/ticket_14297_handle_upstart.rb

```diff
@@ -2,6 +2,8 @@
 
 # only run these on ubuntu vms
 confine :to, :platform => 'ubuntu'
+# vidid and above use systemd rather than upstart
+confine :except, :platform => 'ubuntu-vivid'
```

Comment:
Should we make this a regex so that we don't need to bump this for each ensuing ubuntu release? As the comment says, this is vivid and up.

And ubuntu releases are alphabetical so this might be sth like `/ubuntu-[v-z]/`